app/gateway/changelog.md

```diff
@@ -5,12 +5,301 @@ no_version: true
 ---
 
 ## 2.7.0.0
-**Release Date:** TBD
+**Release Date:** 2021/12/16
 
-#### Deprecated
+### Features
+
+#### Enterprise
+* Kong Gateway now supports installations on [Debian 10 and 11](/gateway/2.7.x/install-and-run/debian/).
+* This release introduces consumer groups, a new entity that lets you
+manage custom rate limiting configuration for any defined subsets of consumers.
+To use consumer groups for rate limiting, configure the [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced)
+plugin with the `enforce_consumer_groups` and `consumer_groups` parameters,
+and use the `/consumer_groups` endpoint to manage the groups.
+  * [Consumer groups reference](/gateway/2.7.x/admin-api/consumer-groups/reference)
+  * [Consumer groups examples](/gateway/2.7.x/admin-api/consumer-groups/examples)
+* The data plane configuration cache can now be encrypted or disabled. Two new
```

Comment:
WDYT about either of these; `disabled` sounds weird to me in this sentence?
```suggestion
* The data plane configuration cache can now be encrypted. Two new
```

or

```suggestion
* The data plane configuration cache can now be encrypted or unencrypted. Two new
```

Reply:
Ignore this; I forgot about the `off` feature with new encryption feature,